<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityTab</class>
 <widget class="QWidget" name="SecurityTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Users and Groups</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="trustee_view">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add_trustee_button">
       <property name="text">
        <string>Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_well_known_trustee_button">
       <property name="text">
        <string>Add well-known trustee...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_trustee_button">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Permissions</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="applied_to_label">
       <property name="text">
        <string>Applied to: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="applied_objects_cmbBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clear_all_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Clear all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="permissions_tab_widget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="common_permissions_tab">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="CommonPermissionsWidget" name="common_permissions_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="extended_permissions_tab">
      <attribute name="title">
       <string>Extended</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="ExtendedPermissionsWidget" name="extended_permissions_widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="creation_deletion_permissions_tab">
      <attribute name="title">
       <string>Creation/deletion</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="CreationDeletionPermissionsWidget" name="creation_deletion_permissions_widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="read_write_permissions_tab">
      <attribute name="title">
       <string>Read/write</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="ReadWritePermissionsWidget" name="read_write_permissions_widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="delegation_tab">
      <attribute name="title">
       <string>Task delegation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="DelegationPermissionsWidget" name="delegation_widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPermissionsWidget</class>
   <extends>QWidget</extends>
   <header>permission_control_widgets/common_permissions_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExtendedPermissionsWidget</class>
   <extends>QWidget</extends>
   <header>permission_control_widgets/extended_permissions_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DelegationPermissionsWidget</class>
   <extends>QWidget</extends>
   <header>permission_control_widgets/delegation_permissions_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreationDeletionPermissionsWidget</class>
   <extends>QWidget</extends>
   <header>permission_control_widgets/creation_deletion_permissions_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReadWritePermissionsWidget</class>
   <extends>QWidget</extends>
   <header>permission_control_widgets/read_write_permissions_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
